docs/classic-ui/icons.md

```diff
@@ -35,99 +35,112 @@ See the file [package.json](https://github.com/plone/plone.staticresources/blob/
 ## Registration
 
 Icons are registered in Plone's registry.
-This provides an option to customize the contenttype and Plone UI icons by overriding icons via XML.
+This provides an option to customize the contenttype and Plone UI icons by overriding icons via XML. Plone ships with the following icon registrations by default.
```

Comment:
When used in narrative text, "contenttype" should be two words, "content type". Else it should be an inline literal or in a code-block.

Also please use one line per sentence.

```suggestion
This provides an option to customize the content type and Plone UI icons by overriding icons via XML.
Plone ships with the following icon registrations by default.
```